### ISSUE TITLE
[mpg123] Build error on arm64-osx

### ISSUE THREAD

**Issue by autoantwort** [microsoft/vcpkg#22206]
Package: mpg123[core]:arm64-osx -> 1.29.2#1
**Host Environment**
- Host: arm64-osx
- Compiler: AppleClang 13.0.0.13000029
-    vcpkg-tool version: 2999-12-31-unknownhash-debug
    vcpkg-scripts version: fb18c46cc 2021-12-24 (2 days ago)

**To Reproduce**
`vcpkg install `

**Failure logs**
```
-- Using cached mpg123-1.29.2.tar.bz2.
-- Cleaning sources at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/mpg123-1.29.2.tar.bz2
-- Applying patch no-executables.patch
-- Applying patch fix-modulejack.patch
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean
-- Configuring arm64-osx-dbg
-- Configuring arm64-osx-rel
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:156 (message):
    Command failed: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:102 (vcpkg_execute_build_process)
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:41 (vcpkg_cmake_build)
  ports/mpg123/portfile.cmake:22 (vcpkg_cmake_install)
  scripts/ports.cmake:145 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/install-arm64-osx-dbg-out.log</summary>

```
[1/57] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -fPIC -MD -MT src/compat/CMakeFiles/compat.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_str.c.o -MF src/compat/CMakeFiles/compat.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_str.c.o.d -o src/compat/CMakeFiles/compat.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_str.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_str.c
[2/57] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -fPIC -MD -MT src/compat/CMakeFiles/compat_dl.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_dl.c.o -MF src/compat/CMakeFiles/compat_dl.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_dl.c.o.d -o src/compat/CMakeFiles/compat_dl.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_dl.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/compat/compat_dl.c
[3/57] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DOPT_GENERIC -DOPT_GENERIC_DITHER -DOPT_MULTI -DOPT_NEON64 -DREAL_IS_FIXED -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/libmpg123/../../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/parse.c.o -MF src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/parse.c.o.d -o src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/parse.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/parse.c
...
Skipped 60 lines
...

/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/optimize.h:198:2: error: "Bad decoder choice together with fixed point math!"
#error "Bad decoder choice together with fixed point math!"
 ^
1 error generated.
[12/57] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DOPT_GENERIC -DOPT_GENERIC_DITHER -DOPT_MULTI -DOPT_NEON64 -DREAL_IS_FIXED -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/libmpg123/../../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o -MF src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o.d -o src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c
FAILED: src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DOPT_GENERIC -DOPT_GENERIC_DITHER -DOPT_MULTI -DOPT_NEON64 -DREAL_IS_FIXED -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/libmpg123/../../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o -MF src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o.d -o src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/readers.c:12:
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/mpg123lib_intern.h:311:
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/optimize.h:198:2: error: "Bad decoder choice together with fixed point math!"
#error "Bad decoder choice together with fixed point math!"
 ^
1 error generated.
[13/57] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DOPT_GENERIC -DOPT_GENERIC_DITHER -DOPT_MULTI -DOPT_NEON64 -DREAL_IS_FIXED -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/libmpg123/../../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o -MF src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o.d -o src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c
FAILED: src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DOPT_GENERIC -DOPT_GENERIC_DITHER -DOPT_MULTI -DOPT_NEON64 -DREAL_IS_FIXED -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/compat -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/arm64-osx-dbg/src/libmpg123 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/libmpg123/../../../../src/libmpg123 -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o -MF src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o.d -o src/libmpg123/CMakeFiles/libmpg123.dir/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c:13:
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/mpg123lib_intern.h:311:
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/optimize.h:198:2: error: "Bad decoder choice together with fixed point math!"
#error "Bad decoder choice together with fixed point math!"
 ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c:847:38: error: use of undeclared identifier 'INT123_synth_1to1_real_neon64'
                fr->synths.plain[r_1to1][f_real] = synth_1to1_real_neon64;
                                                   ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:101:32: note: expanded from macro 'synth_1to1_real_neon64'
#define synth_1to1_real_neon64 INT123_synth_1to1_real_neon64
                               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c:848:39: error: use of undeclared identifier 'INT123_synth_1to1_fltst_neon64'
                fr->synths.stereo[r_1to1][f_real] = synth_1to1_fltst_neon64;
                                                    ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:102:33: note: expanded from macro 'synth_1to1_fltst_neon64'
#define synth_1to1_fltst_neon64 INT123_synth_1to1_fltst_neon64
                                ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c:851:36: error: use of undeclared identifier 'INT123_synth_1to1_s32_neon64'; did you mean 'INT123_synth_1to1_neon64'?
                fr->synths.plain[r_1to1][f_32] = synth_1to1_s32_neon64;
                                                 ^~~~~~~~~~~~~~~~~~~~~
                                                 INT123_synth_1to1_neon64
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:128:31: note: expanded from macro 'synth_1to1_s32_neon64'
#define synth_1to1_s32_neon64 INT123_synth_1to1_s32_neon64
                              ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/decode.h:61:5: note: 'INT123_synth_1to1_neon64' declared here
int synth_1to1_neon64     (real*, int, mpg123_handle*, int);
    ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:53:27: note: expanded from macro 'synth_1to1_neon64'
#define synth_1to1_neon64 INT123_synth_1to1_neon64
                          ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/src/libmpg123/optimize.c:852:37: error: use of undeclared identifier 'INT123_synth_1to1_s32st_neon64'; did you mean 'INT123_synth_1to1_stereo_neon64'?
                fr->synths.stereo[r_1to1][f_32] = synth_1to1_s32st_neon64;
                                                  ^~~~~~~~~~~~~~~~~~~~~~~
                                                  INT123_synth_1to1_stereo_neon64
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:129:33: note: expanded from macro 'synth_1to1_s32st_neon64'
#define synth_1to1_s32st_neon64 INT123_synth_1to1_s32st_neon64
                                ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/libmpg123/decode.h:62:5: note: 'INT123_synth_1to1_stereo_neon64' declared here
int synth_1to1_stereo_neon64(real*, real*, mpg123_handle*);
    ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/mpg123/src/ffb82ffbeb-80fb4e6bd8.clean/ports/cmake/src/../../../src/intsym.h:54:34: note: expanded from macro 'synth_1to1_stereo_neon64'
#define synth_1to1_stereo_neon64 INT123_synth_1to1_stereo_neon64
                                 ^
5 errors generated.
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**
<details><summary>vcpkg.json</summary>

```
{
  "name": "lichtsteuerung",
  "version-string": "2.0.0",
  "dependencies": [
    "gstreamer"
  ]
}

```
</details>

**Comment by JackBoosY** [comment]
Can you please also provide the configure log?

Thanks.

**Comment by autoantwort** [comment]
<details><summary>vcpkg/buildtrees/x264/config-arm64-osx-dbg-out.log</summary>

```
Unknown option --disable-silent-rules, ignored
Unknown option --verbose, ignored
Unknown option --disable-shared, ignored
platform:       X86_64
byte order:     little-endian
system:         MACOSX
cli:            yes
libx264:        internal
shared:         no
static:         yes
bashcompletion: no
asm:            yes
interlaced:     yes
avs:            no
lavf:           no
ffms:           no
mp4:            no
gpl:            yes
thread:         posix
opencl:         yes
filters:        crop select_every
lto:            no
debug:          yes
gprof:          no
strip:          yes
PIC:            no
bit depth:      all
chroma format:  all

You can run 'make' or 'make fprofiled' now.


```
</details>

**Comment by autoantwort** [comment]
@JackBoosY Do you think https://github.com/microsoft/vcpkg-tool/pull/311 should also attach the configure logs always automatically

**Comment by JackBoosY** [comment]
```
system:         MACOSX
```

> @JackBoosY Do you think microsoft/vcpkg-tool#311 should also attach the configure logs always automatically?

That's hard to say:
If we enable this feature by default, it will hurt all CI users.
If we disable this feature by default, it won't work.

**Comment by autoantwort** [comment]
> If we enable this feature by default, it will hurt all CI users.

I don't know why it should hurt all CI users? It only creates a file with a ready to copy issue body

**Comment by JackBoosY** [comment]
> I don't know why it should hurt all CI users? It only creates a file with a ready to copy issue body

At least we should provide the option to disable it, because CI users don't need it and this will increase disk usage.

**Comment by autoantwort** [comment]
> because CI users don't need it and this will increase disk usage.

If I have an error in the ci I would like to report this as well. And all the other files are 15 MB in size (without downloads) while the issue body file is 19 KB small

**Comment by mathisloge** [comment]
> If I have an error in the ci I would like to report this as well 

👍 

CI won't be successful in the normal case anyway, if vcpkg fails. Currently I'm just uploading all vcpkg logs to know what and why it fails. So one file more or less isn't really a issue IMHO

**Comment by autoantwort** [comment]
Fixed by #24161
